packages/plugins/other/visitor-plugin-common/src/base-resolvers-visitor.ts

```diff
@@ -78,6 +80,8 @@ export interface ParsedResolversConfig extends ParsedConfig {
   resolversNonOptionalTypename: ResolversNonOptionalTypenameConfig;
 }
 
+type FieldDefinitionPrintFn = (parentName: string, avoidResolverOptionals: boolean) => string | null;
```

Comment:
I was REALLY confused why/how the field's name is a function. Found out it's because we do custom logic in `FieldDefinition` function. I'm extracting the type here to be able to type it instead of using `any`